src/main/java/de/olivermakesco/polyspring/impl/ToolMaterialInterface.java

```diff
@@ -0,0 +1,7 @@
+package de.olivermakesco.polyspring.impl;
+
+import net.minecraft.world.item.ToolMaterial;
+
+public interface ToolMaterialInterface {
+    ToolMaterial getToolMaterial();
```

Comment:
Since this is being ducked, the method name should be prefixed with `polyspring$` to avoid naming collisions